lib/stimulus_reflex/utils/sanity_checker.rb

```diff
@@ -29,18 +30,22 @@ def called_by_installer?
     def called_by_generate_config?
       ARGV.include? "stimulus_reflex:initializer"
     end
+
+    def called_by_rake?
+      caller.find { |c| c.include?("/gems/rake-") }
```

Comment:
I _think_ this will fail for windows. It appears a more bullet-proof solution is to do this:

```suggestion
      File.basename($PROGRAM_NAME) == 'rake'
```

https://stackoverflow.com/questions/2467208/how-can-i-tell-if-rails-code-is-being-run-via-rake-or-script-generate

Reply:
Is the implication that there's a difference between calling `rake db:migrate` and `rails db:migrate`?